packages/rancher-sriov/templates/crd-template/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+version: 1.0.0
```

Comment:
Version should match v0.1.0 or I think the `catalog.cattle.io/auto-install: sriov-crd=match` annotation will break

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/07398ed56de01f6e7d2a869743c6d5ce8298ca5b/deployment/sriov-network-operator/Chart.yaml#L3